### ISSUE TITLE
Accessibility: low contrast icons in pitch black theme

### ISSUE THREAD

**Issue by davidbtc2009** [nolanlawson/pinafore#2159]
The icons and character count below the compose text box are extremely low contrast when in pitch black theme.

Edit: actually, this applies to the reply, boost, fav, and more icons below posts as well.

**Comment by nolanlawson** [comment]
Some of the themes are contributed, and not all of them are super accessible. Thanks for reporting.

**Comment by asgrim** [comment]
I'd say the contrast isn't great even on the default light theme ("Royal").

![Screenshot from 2022-11-07 16-25-47](https://user-images.githubusercontent.com/496145/200362479-fe4f30e8-3ec9-4f03-af62-4748da878656.png)

When scrolling through a feed, I have to look quite hard to tell if I've starred a post.

**Comment by nolanlawson** [comment]
@asgrim I'm starting to think than an explicit support for high-contrast mode is the best approach.

The CSS `prefers-contrast` did not exist when I first wrote Pinafore. Now that it exists, I should be able to make everyone happier maybe: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast

**Comment by nolanlawson** [comment]
I've also seen folks complain about:

- Gecko theme
- Ozark theme

Certain themes definitely don't have this problem: Cobalt uses a separate color entirely for boost/favorite, and Oaken has a lot of contrast.

I need to find some accessibility guidelines for what the best values here would be. The foreground color also has to be light dark enough to be distinguishable against the background.

Maybe we need some other affordance entirely as an option in the settings – e.g. a prominent border, or a background color, or something like that.

**Comment by nolanlawson** [comment]
Related: #1646 #1831

**Comment by NickColley** [comment]
Consolidating into https://github.com/nolanlawson/pinafore/issues/2230, thank you so much for raising.
